docs/vue-testing-library/api.mdx

```diff
@@ -7,29 +7,27 @@ title: API
 
 It also exposes these methods:
 
-- [`render(Component, options, callback)`](#rendercomponent-options-callback)
+- [`render(Component, options)`](#rendercomponent-options)
   - [Parameters](#parameters)
     - [Component](#component)
     - [Options](#options)
-    - [Callback Function](#callback-function)
   - [`render` result](#render-result)
     - [`...queries`](#queries)
     - [`container`](#container)
     - [`baseElement`](#baseelement)
-    - [`debug(element)`](#debugelement)
-    - [`unmount()`](#unmount)
-    - [`isUnmounted()`](#isunmounted)
-    - [`html()`](#html)
-    - [`emitted()`](#emitted)
-    - [`updateProps(props)`](#updatepropsprops)
+    - [`debug`](#debug)
+    - [`unmount`](#unmount)
+    - [`html`](#html)
+    - [`emitted`](#emitted)
+    - [`rerender`](#rerenderprops)
 - [`fireEvent`](#fireevent)
   - [`touch(elem)`](#touchelem)
   - [`update(elem, value)`](#updateelem-value)
 - [`cleanup`](#cleanup)
 
 ---
 
-## `render(Component, options, callback)`
+## `render(Component, options)`
 
 The `render` function is the only way of rendering components in Vue Testing
 Library.
```

Comment:
there's a sentence below this one that says:

"It takes up to 3 parameters and returns an object with some helper methods."

it should be

"It takes up to 2 parameters and returns an object with some helper methods."